sematic/ui/src/hooks/logHooks.ts

```diff
@@ -7,6 +7,7 @@ import { useLogger } from "../utils";
 import { useHttpClient } from "./httpHooks";
 
 const MAX_LINES = 2000;
+const POLLING_INTERVAL = 5000;
```

Comment:
Can you please edit the PR description to state why this is needed? Is it for responsiveness?

Reply:
@neutralino1 can you help suggest why this is needed? I will summarize it in the PR description.

Reply:
Added a reason being `relive the server from constantly getting polled. `